packages/odyssey-react/src/components/Link/Link.module.scss

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright (c) 2018-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.link {
+  color: cv('blue', '500');
+  text-decoration: none;
+
+  &:hover {
+    color: cv('blue', '500');
+    text-decoration: underline;
+  }
+
+  &:visited {
+    color: cv('blue', '500');
+  }
+
+  /* stylelint-disable selector-no-qualifying-type */
+  &[target='_blank']:not([href^='mailto:'])::after {
+    content: '';
+    display: inline-block;
+    position: relative;
+    inset-block-start: -0.1em;
+    width: 1em;
+    height: 1em;
+    margin-inline-start: 0.2em;
+    background-image: get-icon('external', $color-primary-base);
+    background-repeat: no-repeat;
+    vertical-align: middle;
+  }
+
+  /* stylelint-enable */
```

Comment:
as an aside: if you want to disable a naughty selector as we're doing here you can use

```
/* stylelint-disable-next-line <RULE NAME HERE> */
```

Reply:
Good tip, this was removed as it was no longer necessary!